frontend/src/component/admin/network/NetworkTrafficUsage/chart-functions.ts

```diff
@@ -61,11 +62,14 @@ export const toConnectionChartData = (
                 if (traffic.grouping === 'monthly') {
                     // 1 connections = 7200 * days in month requests per day
                     const daysInMonth = getDaysInMonth(date);
-                    record[dataPoint.period] =
-                        requestCount / (daysInMonth * 7200);
+                    record[dataPoint.period] = Number(
```

Comment:
we keep counting connections based on requests until we get a proper backend model. One change is 1 decimal number instead of 4

Reply:
will add tests in the next PR